src/web/src/components/Header.vue

```diff
@@ -13,7 +13,14 @@
     >
       YACS
     </b-navbar-brand>
-    <b-nav-text class="text-secondary">{{ selectedSemester }}</b-nav-text>
+    <div>
+      <select v-model="selected" v-on:change="selectSemester(selected)">
```

Comment:
Instead of using the `select`, we could use `b-dropdown`. Here's a link to how bootstrap handles the dropdown menu:
[Link](https://bootstrap-vue.org/docs/components/dropdown)

Reply:
Just updated the file to use b-dropdown and added some styling changes as well